spec/unit_tests/stanford/storage_object_validator_spec.rb

```diff
@@ -14,7 +14,7 @@
   describe '#validation_errors' do
     context 'when superclass has validation_errors' do
       it 'returns errors' do
-        expect(error_list.count).to eq(14)
+        expect(error_list.count).to eq(10)
```

Comment:
Why a difference of four? 

1. data dir
2. content dir
3. manifests dir 
4.  metadata dir

All allowed now.